hera_cal/reflections.py

```diff
@@ -579,17 +582,24 @@ def sv_decomp(self, dfft, wgts=None, flags=None, keys=None, Nkeep=None,
 
     def project_svd_modes(self, dfft, umodes=None, svals=None, vmodes=None):
         """
-        Given two of the 3 SVD output vectors, project the input dfft data onto
-        the last remaining SVD vector.
+        Given two of the 3 SVD output matrices, project them onto the input dfft data
+        to estimate the last remaining SVD matrix. Note U and V are unitary matrices.
+
+        If
+            D = U S V
+        then
+            U = D V_dagger S_inv
```

Comment:
In SVD, S is usually not invertible unless D is square.

Reply:
Yeah but for this explanation this is ok. the actual output of `np.linalg` for `S` is a vector of the singular values, such that its inverse is just `1 / S`.

Reply:
I actually think it's confusing. When we mean element-by-element inverse, usually it's written as (S_ij)^-1 (and then you have to do all the index math). At least clarify in the docstring that you mean an element-wise inverse.